woocommerce/woocommerce-android
# Order creation base form

Summary
==========
Partially fixes issue #5225 by defining the Order Creation Fragment, ViewModel, and Navigation graph with the basic model initialization to create an Order from scratch.

⚠️ The Order Status section proper configuration and selection will be tackled in the upcoming PR, I'm making this one available now to unblock the upcoming project tasks.

Screenshots
==========
| Order list FAB interaction to create a full Order  | Inside the Order Creation view |
| ------------- | ------------- |
| ![Screenshot_20211130_041424](https://user-images.githubusercontent.com/5920403/144004347-bfe8ae94-76cf-4dff-b500-1f69d9931617.png) | ![Screenshot_20211130_042848](https://user-images.githubusercontent.com/5920403/144004368-12b3f200-bf36-4b8c-8e68-222b44410712.png) |

How to Test
==========
1. Go to the Order List view and hit the FAB at the lower right corner
2. If Simple Payments feature is disabled, the click should redirect to the new Order Creation view immediately, if not, the Bottom sheet with the Order Creation / Simple Payment should be displayed, select the Order Creation option
3. Verify that the navigation to the Order Creation view works as expected, and the back button navigate back to the Order list as expected


Update release notes:
